src/core/analytics.js

```diff
@@ -392,16 +392,19 @@ class Analytics {
                 let transformedEvents;
                 if (transformationServerAccess) {
                   // filter the transformed event for that destination
-                  transformedEvents = transformedPayload.find(
+                  const filteredDestination = transformedPayload.find(
                     (e) => e.id === intg.destinationId,
-                  ).payload;
+                  );
+                  transformedEvents = filteredDestination ? filteredDestination.payload : undefined;
```

Comment:
```suggestion
                  transformedEvents = transformedPayload.find(
                    (e) => e.id === intg.destinationId,
                  )?.payload;
```

Reply:
This is `analytics.js` file, AFAIR this doesn't work.

Reply:
Done